src/components/CardBody/CardBody.tsx

```diff
@@ -15,8 +15,7 @@ export interface Props {
 
 /**
  * Primary UI component for user interaction
- * BETA: This component is still a work in progress and is subject to change.
- *
+
```

Comment:
Slight find and replace shenanigan (wow, I don't think I've ever used the singular form of shenanigans before 😂)

```suggestion
```

Reply:
Whoops, thanks!